hooks/VDM.js

```diff
@@ -305,303 +305,229 @@ module.exports = function (input, session) {
   }
 
   if (data.lon && data.lat) {
-    meteoLocation = data.lon.toString().replace('.', '_') + "__" + data.lat.toString().replace('.', '_')
+    values.push({
+      path: 'navigation.position',
+      value: {
+        longitude: data.lon,
+        latitude: data.lat,
+      },
+    })
   }
 
   if (data.avgwindspd) {
     contextPrefix = 'meteo.'
-    if (data.avgwindspd < 127) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.windAverageSpeed`,
-        value: utils.transform(data.avgwindspd, 'knots', 'ms'),
-      })
-    } 
+    values.push({
+      path: `environment.air.windAverageSpeed`,
+      value: utils.transform(data.avgwindspd, 'knots', 'ms'),
+    })
   }
 
   if (data.windgust) {
     contextPrefix = 'meteo.'
-    if (data.windgust < 127) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.windGust`,
-        value: utils.transform(data.windgust, 'knots', 'ms'),
-      })
-    }
+    values.push({
+      path: `environment.air.windGust`,
+      value: utils.transform(data.windgust, 'knots', 'ms'),
+    })
   }
 
   if (data.winddir) {
     contextPrefix = 'meteo.'
-    if (data.winddir < 360) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.windDirection`,
-        value: utils.transform(data.winddir, 'deg', 'rad'),
-      })
-    }
+    values.push({
+      path: `environment.air.windDirection`,
+      value: utils.transform(data.winddir, 'deg', 'rad'),
+    })
   }
 
   if (data.windgustdir) {
     contextPrefix = 'meteo.'
-    if (data.windgustdir < 360) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.windGustDirection`,
-        value: utils.transform(data.windgustdir, 'deg', 'rad'),
-      })
-    }
+    values.push({
+      path: `environment.air.windGustDirection`,
+      value: utils.transform(data.windgustdir, 'deg', 'rad'),
+    })
   }
 
   if (data.airtemp) {
     contextPrefix = 'meteo.'
-    if (data.airtemp > -601 && data.airtemp < 601) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.temperature`,
-        value: utils.transform((data.airtemp / 10), 'c', 'k'),
-      })
-    }
+    values.push({
+      path: `environment.air.temperature`,
+      value: utils.transform(data.airtemp, 'c', 'k'),
+    })
   }
 
   if (data.relhumid) {
     contextPrefix = 'meteo.'
-    if (data.relhumid < 101) {
-      values.push({
-        path: `environment.observations.${meteoLocation}.air.relativeHumidity`,
-        value: data.relhumid,
-      })
-    }
+    values.push({
+      path: `environment.air.relativeHumidity`,
```

Comment:
To align with `environment` group in the spec should we use `environment.observations.outside` instead of `environment.observations.air`?